interface/wx/dirctrl.h

```diff
@@ -212,6 +240,22 @@ class wxGenericDirCtrl : public wxControl
     */
     virtual void ReCreateTree();
 
+    /**
+        Add an item to the popup menu. This function should only be called with the wxEVT_DIRCTRL_MENU_POPPED_UP event handler.
+        Returns the ID of the new menu item.
+        
+        See the widgets sample for a full example of the popup menu.
+        \code{.cpp}
+        void MyFrame::DirectoryMenuPopped(wxCommandEvent &evt)
+        {
+            m_dirCtrl->GetMenu()->AppendSeparator();
+            int id = m_dirCtrl->NewMenuItem(wxT("Directory Properties"));
+            m_dirCtrl->Bind(wxEVT_MENU, &MyFrame::DirProperties, this, id);
+        }
+        \endcode
+    */
+    int NewMenuItem(const wxString& label);
```

Comment:
Sorry but I don't think it's a good API. We already have methods in `wxMenu` for appending items, separators, check and radio items and so on. This method is more limited, so even if you don't need neither check nor radio items, you still need to use the menu directly to append a separator. So why have it at all? Just `GetMenu()` is enough, AFAICS.

Am I missing anything? If not, let's please remove this one.